### PR TITLE
[SK] Cleanup and more possibilities

### PR DESCRIPTION
**Pull request by Pheggas**
### Summary
- Quick cleanup of `_common.yaml`, following order and sentence type in EN language of `_common.yaml`. It is more pleasant and polite to users now.
- Added possibility of specifying area in turn on / off sentences. (**Please check if syntax used is valid**)

## Summary by CodeRabbit

- **New Features**
  - Expanded error messages for clearer user feedback, including new responses for various scenarios.
  - Enhanced command flexibility by allowing area specifications in device control commands.

- **Bug Fixes**
  - Updated existing error messages for improved clarity and specificity.

- **Documentation**
  - Modifications to intent definitions to accommodate optional area context in commands.

- **Improvements**
  - Added new sections and values for temperature units and brightness levels, enhancing user interaction vocabulary.

### CODE REVIEW

File: sentences/sk/_common.yaml [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
+    no_entity_in_floor_exposed: "Prepáč, {{ entity }} na poschodí {{ floor }} nie je zverejnená"
+    no_domain_exposed: "Prepáč, žiadna {{ domain }} nie je zverejnená"
+    no_domain_in_area_exposed: "Prepáč, žiadna {{ domain }} v oblasti {{ area }} nie je zverejnená"
+    no_domain_in_floor_exposed: "Prepáč, žiadna {{ domain }} na poschodí {{ floor }} nie je zverejnená"
+    no_device_class_exposed: "Prepáč, žiadna {{ device_class }} nie je zverejnená"
+    no_device_class_in_area_exposed: "Prepáč, žiadna {{ device_class }} v oblasti {{ area }} nie je zverejnená"
+    no_device_class_in_floor_exposed: "Prepáč, žiadna {{ device_class }} na poschodí {{ floor }} nie je zverejnená"
+
+    duplicate_entities: "Prepáč, existuje viacero zariadení menom {{ entity }}"
+    duplicate_entities_in_area: "Sorry, existuje viacero zariadení menom {{ entity }} v oblasti {{ area }}"
```
Comment (LubosKadasi) [inline sentences/sk/_common.yaml:32]: "Prepáč ..."

File: sentences/sk/_common.yaml [hunk 2]
```diff
@@ -89,9 +128,9 @@ expansion_rules:
   turn_off: "(vypni|vypnúť)"
   turn_on_light: "(zasvieť|zažni|rozžni|rozsvieť)"
   turn_off_light: "(zahas|zhasni)"
-  set: "(nastav|prestav|zmeň|zvýš|zníž)"
-  open: "(otvor|odostri|(roz|vy)tiahni)"
-  close: "(zatvor|zavri|zastri|(s|za)tiahni)"
+  set: "(nastav[iť]|prestav[iť]|zme([ň]|niť)|zvýš[iť]|zníž[iť])"
```
Comment (LubosKadasi) [inline sentences/sk/_common.yaml:131]: zme(ň|iť) ?

File: sentences/sk/_common.yaml [hunk 3]
```diff
@@ -89,9 +128,9 @@ expansion_rules:
...
+  open: "(otvor[iť]|odostri[eť]|(roz|vy)tiahn[i|uť])"
+  close: "(zatvor[iť]|zavri[eť]|zatvor[iť]|zastri[eť]|(s|za)tiahn([i]|[uť]))"
```
Comment (LubosKadasi) [inline sentences/sk/_common.yaml:133]: (s|za)tiahn(i|uť)

**Review by LubosKadasi — COMMENTED** [review]
👍 THX

File: sentences/sk/_common.yaml [hunk 4]
```diff
@@ -1,12 +1,41 @@
...
+    no_entity_in_area_exposed: "Prepáč, {{ entity }} v oblasti {{ area }} nie je zverejnená"
+    no_entity_in_floor_exposed: "Prepáč, {{ entity }} na poschodí {{ floor }} nie je zverejnená"
+    no_domain_exposed: "Prepáč, žiadna {{ domain }} nie je zverejnená"
+    no_domain_in_area_exposed: "Prepáč, žiadna {{ domain }} v oblasti {{ area }} nie je zverejnená"
+    no_domain_in_floor_exposed: "Prepáč, žiadna {{ domain }} na poschodí {{ floor }} nie je zverejnená"
+    no_device_class_exposed: "Prepáč, žiadna {{ device_class }} nie je zverejnená"
+    no_device_class_in_area_exposed: "Prepáč, žiadna {{ device_class }} v oblasti {{ area }} nie je zverejnená"
+    no_device_class_in_floor_exposed: "Prepáč, žiadna {{ device_class }} na poschodí {{ floor }} nie je zverejnená"
+
+    duplicate_entities: "Prepáč, existuje viacero zariadení menom {{ entity }}"
```
[the remark this answers is not in the document]
Comment (LubosKadasi) [reply sentences/sk/_common.yaml:31]: "s názvom"

File: sentences/sk/_common.yaml (hunk 2 above)
Comment (Pheggas) [reply sentences/sk/_common.yaml:131]: Ak by bolo zme(ň|iť) tak by vypísalo zmeiť (n by tam chýbalo)

Comment (LubosKadasi) [reply sentences/sk/_common.yaml:131]: Čiže "zme(ň|niť)" lebo tieto možnosti nie sú optional ale alternative

Comment (Pheggas) [reply sentences/sk/_common.yaml:131]: ah, right. Prepáč nevšimol som si :D

**Comment by Pheggas** [comment]
Spravil som úpravy tak prosím last check 👍

File: sentences/sk/_common.yaml [hunk 5]
```diff
@@ -1,12 +1,41 @@
...
+    no_entity_in_floor_exposed: "Prepáč, {{ entity }} na poschodí {{ floor }} nie je zverejnená"
+    no_domain_exposed: "Prepáč, žiadna {{ domain }} nie je zverejnená"
+    no_domain_in_area_exposed: "Prepáč, žiadna {{ domain }} v oblasti {{ area }} nie je zverejnená"
+    no_domain_in_floor_exposed: "Prepáč, žiadna {{ domain }} na poschodí {{ floor }} nie je zverejnená"
+    no_device_class_exposed: "Prepáč, žiadna {{ device_class }} nie je zverejnená"
+    no_device_class_in_area_exposed: "Prepáč, žiadna {{ device_class }} v oblasti {{ area }} nie je zverejnená"
+    no_device_class_in_floor_exposed: "Prepáč, žiadna {{ device_class }} na poschodí {{ floor }} nie je zverejnená"
+
+    duplicate_entities: "Prepáč, existuje viacero zariadení s názvom {{ entity }}"
+    duplicate_entities_in_area: "Prepáč, existuje viacero zariadení názvom {{ entity }} v oblasti {{ area }}"
```
Comment (LubosKadasi) [inline sentences/sk/_common.yaml:32]: s názvom

File: sentences/sk/_common.yaml [hunk 6]
```diff
@@ -1,12 +1,41 @@
...
+    duplicate_entities_in_floor: "Prepáč, existuje viacero zariadení názvom {{ entity }} na poschodí {{ floor }}"
```
Comment (LubosKadasi) [inline sentences/sk/_common.yaml:33]: s názvom

**Review by LubosKadasi — APPROVED** [review]
inak super a ďakujem

**Review by LubosKadasi — APPROVED** [review]

**Comment by Pheggas** [comment]
To je fajn no že nejde buildnuť 😄 to bude musieť asi niekto force mergnuť

**Comment by LubosKadasi** [comment]
> To je fajn no že nejde buildnuť 😄 to bude musieť asi niekto force mergnuť

Vyzeralo to, že je tam nejaký zvláštny whitespace. Už by to malo byť všetko ok.

**Comment by Pheggas** [comment]
aha, sorry a ďakujem. Pravdepodobne budem aj naďalej prispievať :)

**Pull request merged**